packages/safe-ds-vscode/src/extension/pythonServer.ts

```diff
@@ -435,7 +406,11 @@ const connectToWebSocket = async function (): Promise<void> {
                     logOutput(`[Runner] Message received: (${event.type}, ${typeof event.data}) ${event.data}`);
                     return;
                 }
-                logOutput(`[Runner] Message received: '${event.data}'`);
+                logOutput(
```

Comment:
@SmiteDeluxe According to my tests, this change fixes the unresponsiveness after loading the large table. Please test, if this is the case for you.

Reply:
I'll merge it now, we've waited long enough for a response.